packages/cloudflare-workers-bindings-extension/src/show-bindings.ts

```diff
@@ -198,6 +497,65 @@ export class BindingsProvider implements vscode.TreeDataProvider<Node> {
 	}
 }
 
+export function getBindingName(type: BindingType): string {
```

Comment:
nit: could be a record like [`friendlyBindingNames`](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/utils/print-bindings.ts) in here? 
(very much personal preference)

Reply:
[54db155](https://github.com/cloudflare/workers-sdk/pull/7597/commits/54db155295c182ea0c22822bbe82b552c5482927)